File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Prompt user for height
    int h;
    do
    {
        h = get_int("Positive integer less than 23: ");
    }
    while (h < 0 || h > 23);

    // Steps for drawing each row of the pyramid
    // Print out configured rows until i is not < h
    for (int i = 0; i < h; i++)
    {
        // print spaces for left pyramid
        for (int j = 0; j < h - (i + 1); j++)
        {
            printf(" ");
        }

        // print hashes for left pyramid
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // print gap
        for (int j = 0; j < 2; j++)
        {
            printf(" ");
        }
        // print hashes for right pyramid
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // print new line
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 0 and height <= 23:
        break

# for each row
for i in range(height):

    # print spaces for left pyramid
    for j in range(height - (i + 1)):
        print(" ", end = "")

    # print hashes for left pyramid
    for j in range(i + 1):
        print("#", end = "")

    # print gap
    for j in range(2):
        print(" ", end = "")

    # print hashes for right pyramid
    for j in range(i + 1):
        print("#", end = "")

    # print new line
    print()
